Sources/CXTest/TransparentPublisher.swift

```diff
@@ -0,0 +1,22 @@
+import CXShim
+
+extension AnyPublisher {
+    
+    public init(_ receiveSubscriber: @escaping (AnySubscriber<Output, Failure>) -> Void) {
```

Comment:
I believe this suit for AnyPublisher very well. So we don't need to expose TransparentPublisher.